Mathlib/RingTheory/Ideal/Quotient.lean

```diff
@@ -111,14 +113,17 @@ protected theorem eq : mk I x = mk I y ↔ x - y ∈ I :=
 #align ideal.quotient.eq Ideal.Quotient.eq
 
 @[simp]
-theorem mk_eq_mk (x : R) : (Submodule.Quotient.mk x : R ⧸ I) = mk I x :=
-  rfl
+theorem mk_eq_mk (x : R) : (Submodule.Quotient.mk x : R ⧸ I) = mk I x := rfl
 #align ideal.quotient.mk_eq_mk Ideal.Quotient.mk_eq_mk
 
 theorem eq_zero_iff_mem {I : Ideal R} : mk I a = 0 ↔ a ∈ I :=
   Submodule.Quotient.mk_eq_zero _
 #align ideal.quotient.eq_zero_iff_mem Ideal.Quotient.eq_zero_iff_mem
 
+-- Porting note: new theorem
+theorem mk_eq_mk_iff_sub_mem (x y : R) : mk I x = mk I y ↔ x - y ∈ I := by
```

Comment:
@jcommelin do you remember why you added this, it looks the same as Ideal.Quotient.eq with different implicitness of arguments?